stdlib/LinearAlgebra/test/eigen.jl

```diff
@@ -132,4 +132,11 @@ end
     @test factstring == "$(summary(e))\neigenvalues:\n$valsstring\neigenvectors:\n$vecsstring"
 end
 
+@testset "eigen of an Adjoint" begin
+    A = randn(3,3)
+    @test eigvals(A') == eigvals(copy(A'))
+    @test eigen(A')   == eigen(copy(A'))
+end
```

Comment:
Is it worth explicity testing `eigmin` and `eigmax` too?
```suggestion
    @test eigmin(A') == eigmin(copy(A'))
    @test eigmax(A') == eigmax(copy(A'))
end
```